examples/android/app/build.gradle

```diff
@@ -37,16 +37,18 @@ protobuf {
     generateProtoTasks {
         all().each { task ->
             task.builtins {
-                javanano {
-                    // Options added to --javanano_out
-                    option 'ignore_services=true'
+                // Javanano is installed by default, so we need to remove it like so
```

Comment:
Very minor nit -- do we need to remove nano to use lite? If so, we should change this comment to indicate that eg: 
// Javanano is installed by default, but needs to be removed to use protobuf lite..